modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/GenerateTestrayCSVUtil.java

```diff
@@ -112,62 +113,126 @@ public static void generate(
 		}
 	}
 
-	private static int _getRecentFailures(
-		JSONObject resultJSONObject, int casesChecked) {
+	private static boolean _areSimilarErrors(String error1, String error2) {
+		Double distance = StringUtils.getJaroWinklerDistance(error1, error2);
 
-		try {
-			JSONObject historyJSONObject =
-				JenkinsResultsParserUtil.toJSONObject(
-					resultJSONObject.getString("htmlURL") + "/history.json");
+		if (distance > 0.8) {
+			return true;
+		}
+
+		return false;
+	}
```

Comment:
@kyle-miho According to Wikipedia:

`The higher the Jaro–Winkler distance for two strings is, the less similar the strings are. The score is normalized such that 0 means an exact match and 1 means there is no similarity. The original paper actually defined the metric in terms of similarity, so the distance is defined as the inversion of that value (distance = 1 − similarity).`

Wouldn't this mean that the errors are simliar if distance < 0.8? Not  > 0.8?